sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/configuration/CompatibilityNotMetException.java

```diff
@@ -0,0 +1,13 @@
+package com.azure.spring.cloud.autoconfigure.configuration;
```

Comment:
How about putting them in the package `com.azure.spring.cloud.autoconfigure.compatibility`?